Runtime/Game/Resources/LootLockerConfig.cs

```diff
@@ -84,7 +102,7 @@ public enum platformType { Android, iOS, Steam, PlayStationNetwork }
         [HideInInspector]
         public string baseUrl = "https://api.lootlocker.io";
         public enum DebugLevel { All, ErrorOnly, NormalOnly, Off }
-        public DebugLevel currentDebugLevel;
+        public DebugLevel currentDebugLevel = DebugLevel.All;
```

Comment:
Not sure where this is used, but I don't think we want to enable debug logging for people?

Reply:
Since we are now creating a new LootLocker-config instead of using the one that existed in the repository, the debug-level will be set to DebugLevel.Off, which means that you will not see any messages from LootLocker in the Unity Console. The previous "already existing" Config-asset had this set to DebugLevel.All.